ssh/tailssh/tailssh_test.go

```diff
@@ -210,7 +210,7 @@ func TestMatchRule(t *testing.T) {
 				info: tt.ci,
 				srv:  &server{logf: t.Logf},
 			}
-			got, gotUser, err := c.matchRule(tt.rule, nil)
+			got, gotUser, _, err := c.matchRule(tt.rule, nil)
 			if err != tt.wantErr {
 				t.Errorf("err = %v; want %v", err, tt.wantErr)
 			}
```

Comment:
> in particular cases where multiple matching rules have differing acceptEnv configurations.

Are you thinking we'd want to add a test for `evalSSHPolicy` with a policy that has multiple rules, and multiple that would match, and assert that we get the `acceptEnv` for the first match out?

Reply:
Yes. Sorry, I hadn't realized that this test is only matching single rules and that we're not (yet) testing evalSSHPolicy.

The whole "we match the first rule" piece of logic seems pretty important and worth encoding in a test to avoid regressing.

Reply:
Oh yeah no worries! Yeah makes sense to me, will add a test.